lms/djangoapps/courseware/date_summary.py

```diff
@@ -107,7 +108,7 @@ def _format_date(self):
         # 'absolute'. For example, 'absolute' might be "Jan 01, 2020",
         # and if today were December 5th, 2020, 'relative' would be "1
         # month".
-        date_format = _("{relative} ago - {absolute}") if date_has_passed else _("in {relative} - {absolute}")
+        date_format = _(u"{relative} ago - {absolute}") if date_has_passed else _(u"in {relative} - {absolute}")
```

Comment:
This kind of pasting together of sentences is almost certainly impossible to translate properly.  We don't even have translators comments here explaining what these strings mean.  But I'm not sure how much to pull on that thread.